cypress.json

```diff
@@ -7,5 +7,6 @@
     "runMode": 2,
     "openMode": 0
   },
-  "blockHosts": "*.google-analytics.com"
+  "blockHosts": "*.google-analytics.com",
+  "nodeVersion": "system"
```

Comment:
The default sets the node version to the package that is bundled with Cypress. [See here for more info](https://docs.cypress.io/guides/references/configuration#Node-version)

This means that we were not necessarily running our tests with Node v12 as specified in our workflow. This instructs Cypress to use the version on the CI machine(s) (or local machine).

Reply:
Those docs state the default value for `nodeVersion` is `system`. You're just resetting the value to be the default value. You can remove it

Reply:
Also states its deprecated and will be removed 
![image](https://user-images.githubusercontent.com/368249/143091931-726e3bda-9826-4b0d-9959-e8c71e3aefd4.png)

Reply:
@SiTaggart I'm going to remove this for now, but FWIW added [this line to print the node version cypress is using](https://github.com/twilio-labs/paste/pull/2063/files#diff-9079843739331d8f7ae3e87cd86d1711e6a9fc3d30730886dd06d44d95808df5R493) it printed out the version that is bundled with the cypress version.

I wonder if the above snippet from the docs is true for more recent versions of cypress, but we are currently a few major version behind.